app/assets/v2/js/pages/new_bounty.js

```diff
@@ -319,6 +339,16 @@ $(document).ready(function() {
         txid: null
       });
 
+      web3.eth.sendTransaction({
+        to: GITCOIN_TIP_ADDRESS,
+        value: gitcoinTipAmount * decimalDivisor,
+        gasPrice: web3.toHex($('#gasPrice').val() * Math.pow(10, 9)),
+        gas: web3.toHex(318730),
+        gasLimit: web3.toHex(318730)
```

Comment:
Should we pass these values in dynamically?

Reply:
Any word?

Reply:
Agreed

Reply:
I'm using the same method that the actual price for the bounty takes. Do you want me to change this in both places? What do you mean by dynamic? As in the user enters the values?

Reply:
Still need some insight on this, then we're done

Reply:
`web3.eth.estimateGas()` should get you the estimated gas cost of this tx.  i'd suggest using that

Reply:
So I should use `web3.eth.estimateGas` for the `gas` and `gasLimit` properties? Should I leave the `gasPrice` property value to be obtained from the client?

Reply:
Take a look now, I'm only getting the `gas` and `gasLimit` dynamically for the Gitcoin tip transaction.

Reply:
If there is an error while executing `estimateGas`, the funding transaction will still go through. Is that okay?

Reply:
What are the values that go through in case of an error?

Reply:
I believe everything besides the tip will go through if there is an error. So if the tip errors, the top bar will indicate an error and obviously the transaction will not go through, but the issue will still be funded because the main transaction will still be executed, unless of course the main transaction also has an error